R/compute_kidney.R

```diff
@@ -0,0 +1,163 @@
+#' Compute Estimated Glomerular Filtration Rate (eGFR) for Kidney Function
+#'
+#' Compute Kidney Function Tests:
+#'   Estimated Creatinine Clearance (CRCL) by Cockcroft-Gault equation
+#'   Estimated Glomerular Filtration Rate (eGFR) by CKD-EPI or MDRD equations
+#'
+#' @param creat Creatinine (umol/L)
```

Comment:
Keep having discussion on this issue

Reply:
(umol/L) is correct (SI unit), however we could check units for mg/dL.  Currently umol/L is assumed and is converted to mg/dL in the CKD-EPI and MDRD equations.

Reply:
oh lol this was just a demo of the PR potential.  I wasn't questioning this, but maybe this stirred your interest in some checks! :)

Reply:
Should we add another parameter for a Creatinine Unit variable (ex: creatu)?  Then there could be conditional creat conversions as-needed for the different methods.  Then would eliminate any unit uncertainty when passing in 'creat'?

Reply:
Added creatu parameter @KDpharma.  It will accept values "SI" and "CV" in addition to "umol/L" and "mg/dL".  It will default to "SI" and can be a single string or variable.